pkg/getter/registrygetter.go

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
perhaps this package can be moved into internal with the rest of the registry stuff

Reply:
see the TODO for this – I'm intending to just drop this file entirely and make the internal getter.go hold all this. The reason it was done this way originally was because of an import loop, but that's since been resolved